cves/kernel/CVE-2014-0101.yml

```diff
@@ -167,8 +151,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: Automated tools like fuzzers can be highly effective at discovering vulnerabilities like CVE-2014-0101. Fuzzers generate a large number of test inputs, including potentially malformed or malicious inputs,to see how the software handles them. A fuzzer or similar automated tool could send a variety of SCTP packets,including ones with specially crafted AUTH chunks, to test the SCTP implementation. If the tool discovers that the system crashes in response to certain inputs, it can flag the issue as a potential vulnerability so it is plausible that an automated tool could have discovered this.
```

Comment:
Seems like I could crash every outdated linux system connected to the internet using this.

Reply:
You aren't wrong but at the same time, I think it would be very hard to find a linux system that has a version form over a decade ago still running